.github/workflows/_releases.yml

```diff
@@ -0,0 +1,137 @@
+name: Releases
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+permissions:
+  contents: write
+
+jobs:
+
+  linters:
+    uses: ./.github/workflows/linters.yml
+
+  tests-standard:
+    uses: ./.github/workflows/tests-standard.yml
+
+  tests-premium:
+    uses: ./.github/workflows/tests-premium.yml
+
+  publish-docs:
+    needs:
+      - linters
+      - tests-standard
+      - tests-premium
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.12
+      - name: Install dependencies
+        run: |
+          pip install -e .[docs]
+      - name: Build docs
+        run: |
+          mkdocs build
+      - name: Deploy 🚀
+        uses: JamesIves/github-pages-deploy-action@v4.4.1
+        with:
+          branch: gh-pages
+          folder: site
+
+  publish-to-github:
+    needs:
+      - linters
+      - tests-standard
+      - tests-premium
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create release in GitHub
+        uses: ncipollo/release-action@v1
+        with:
+          name: ${{ github.ref }}
+          body: "(this is a placeholder, it will be replaced with a changelog contents)"
+          # This token is provided by Actions, you do not need to create your own token
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+  publish-to-pypi:
+    needs:
+      - linters
+      - tests-standard
+      - tests-premium
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.12
+      - name: Build
+        run: |
+          pip install wheel setuptools
+          python setup.py build sdist bdist_wheel
+      - name: Publish distribution 📦 to PyPI
+        uses: pypa/gh-action-pypi-publish@v1.5.0
+        with:
+          password: ${{ secrets.PYPI_API_TOKEN }}
+
+  publish-to-dockerhub:
```

Comment:
I'm unable to find the publish to dockerhub. There is a login to ghcr on line 122.
I can't see other registries in "Publish image to registries" job.

Reply:
True, we use GHCR. Corrected the name.